src/e2e/config/webdriverio.config.js

```diff
@@ -17,8 +20,8 @@ const config = {
     unicodeKeyboard: true,
     build: process.env.BROWSERSTACK_BUILD_NAME || `Local - ${user} - ${new Date().toISOString().slice(0, 10)}`,
     project: process.env.BROWSERSTACK_PROJECT_NAME || 'em',
-    'browserstack.localIdentifier': process.env.BROWSERSTACK_LOCAL_IDENTIFIER || 'local-' + new Date().getTime(),
```

Comment:
It looks like we are not using `BROWSERSTACK_LOCAL_IDENTIFIER` at all any longer. What was the purpose of `BROWSERSTACK_LOCAL_IDENTIFIER` and what is the impact of removing it?

https://docs.travis-ci.com/user/browserstack/#local-identifier

Reply:
A local identifier is a unique id that identifies a local connection to the browser stack. If we are using tests in sequential order it is fine to have a single local identifier, which was actually the purpose of the `BROWSERSTACK_LOCAL_IDENTIFIER`. Since we are running the parallelized tests in jests, there were multiple connections to the browser stack that were probably causing the intermittent failure. So I removed the use of a single `BROWSER_STACK_LOCAL_IDENTIFIER` and used random id instead. If we were to run tests in sequential order it would make sense to have a single `BROWSER_STACK_LOCAL_IDENTIFIER`.

Reply:
That makes sense. Why was it stored as an environment variable though?

Reply:
> That makes sense. Why was it stored as an environment variable though?

I have no idea. I think it was supposed to be changed on development so that devs working on it can check their browser stacks logs based on that identifier.